### ISSUE TITLE
Colormap test errors for new matplotlib

### ISSUE THREAD

**Issue by matthew-brett** [nipy/dipy#1188]
Test error when testing against latest matplotlib:

```
======================================================================
FAIL: dipy.viz.tests.test_fvtk.test_colormaps_matplotlib
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/matthew-brett/dipy/venv/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/build/matthew-brett/dipy/venv/lib/python2.7/site-packages/numpy/testing/decorators.py", line 147, in skipper_func
    return f(*args, **kwargs)
  File "/home/travis/build/matthew-brett/dipy/venv/lib/python2.7/site-packages/dipy/viz/tests/test_fvtk.py", line 118, in test_colormaps_matplotlib
    npt.assert_array_almost_equal(rgba1, rgba2, 1)
  File "/home/travis/build/matthew-brett/dipy/venv/lib/python2.7/site-packages/numpy/testing/utils.py", line 979, in assert_array_almost_equal
    precision=decimal)
  File "/home/travis/build/matthew-brett/dipy/venv/lib/python2.7/site-packages/numpy/testing/utils.py", line 796, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 1 decimals
(mismatch 16.675%)
 x: array([[ 1. ,  1. ,  0.6,  1. ],
       [ 0.2,  0.4,  0.7,  1. ],
       [ 0.7,  0.7,  0.8,  1. ],...
 y: array([[ 0.8,  0.8,  0.6,  1. ],
       [ 0.6,  0.7,  0.6,  1. ],
       [ 0.8,  0.7,  0.7,  1. ],...
```

https://travis-ci.org/matthew-brett/dipy/jobs/208424940

I'm guessing this is because of the new default 'viridis' colormap in matplotlib >= 2.0.

**Comment by matthew-brett** [comment]
Cross ref new testing regime at #1186 .

**Comment by arokem** [comment]
Strangely enough, this is happening with the "Accent" color map. Looks like that color-map might have changed in Matplotlib. We might need to change the values we have in: 

https://github.com/nipy/dipy/blob/master/dipy/data/files/dipy_colormaps.json

@MrBago : how were those values generated? IIRC, you created that file.

**Comment by matthew-brett** [comment]
The Accent colormap did change, essentially from a continuous to a discrete map - see:
http://matplotlib.org/users/whats_new.html#qualitative-colormaps.

Maybe it's worth deprecating the `Accent` colormap name in dipy, and renaming or removing?

**Comment by arokem** [comment]
This was resolved in #1193
